src/components/Messages.js

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { fetchMessages } from '../redux/messages/messageReducer';
+import Message from './Message';
+
+const Messages = () => {
+  const messages = useSelector((state) => state.messageReducer.messages);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    if (messages.length < 5) {
+      dispatch(fetchMessages());
+    }
+  }, [dispatch, messages.length]);
```

Comment:
- [ ] You did a good job working on the previous reviewer's suggestions. 💯 However, according to the previous reviewer and the project requirements, you should only display `one` random message. Please consider fixing this! 🤓 🔧